portfolio/src/main/webapp/comments.html

```diff
@@ -1,54 +1,62 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Comments</title>
-    <link rel="stylesheet" href="style.css">
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300&family=Poiret+One&family=Roboto&display=swap" rel="stylesheet">
-    <script src="script.js"></script>
-  </head>
-  <body onload="getComments()">
-    <div id="content">
-      <header>
-        <h1>Rachel's Portfolio</h1>
-      </header>
-      <div id="nav">
-        <ul>
-          <li><a href="index.html">Home</a></li>
-          <li><a href="gallery.html">Gallery</a></li>
-          <li><a class="active" href="comments.html">Comments</a></li>
-        </ul>
-      </div>
-      
+
+<head>
+  <meta charset="UTF-8">
+  <title>Comments</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+
+<body onload="getComments()">
+  <div id="content">
+    <header>
+      <h1>Rachel's Portfolio</h1>
+    </header>
+    <div id="nav">
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="gallery.html">Gallery</a></li>
+        <li><a class="active" href="comments.html">Comments</a></li>
+      </ul>
+    </div>
+    <select onchange="getComments()" name="language-dropdown" id="language-dropdown">
+      <option value="zh">Chinese</option>
```

Comment:
A hardcoded list of options is a good start; the API supports the ability to query for the list of supported languages, you could use that to populate this instead.

Reply:
Ah nice I'll look into it. Would a sensible way be to write a js function that runs when the page loads?

Reply:
Yep, function called on load that populates the values would be good option.